src/task2.js

```diff
@@ -0,0 +1,45 @@
+const csv = require('csvtojson');
+const fs = require('fs');
+
+const writeText = fs.createWriteStream('node_mentoring_t1_2_input_example.txt');
+const results = [];
+
+var s = fs.createReadStream('node_mentoring_t1_2_input_example.csv')
+        .pipe(csv())
+        .on('data', (data) =>  writeText.write(data))
+        .on('error', function(err){
```

Comment:
Consider storing your error handling utilities separately in a "utils" folder in your root "src".